test/staking/delegation.test.ts

```diff
@@ -593,68 +594,6 @@ describe('Staking::Delegation', () => {
       expect(await staking.getIndexerCapacity(indexer.address)).eq(0)
     })
 
-    it('should send delegation cut of query fees to delegation pool', async function () {
```

Comment:
I'm curious why this test was removed, is this behavior not the same anymore?

Reply:
It is. I moved this to `allocation.test.ts`, the function `shouldCollect` tests for the same things but in a lot more scenarios.

Reply:
Aaah makes sense, missed that, thanks